packages/Checkbox/src/Checkbox.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuid from "uuid/v4";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import CheckIcon from "@paprika/icon/lib/Check";
+import AddIcon from "@paprika/icon/lib/Add";
+import checkboxStyles from "./Checkbox.styles";
+
+const propTypes = {
+  a11yText: PropTypes.string,
+  children: PropTypes.node,
+  isChecked: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  isIndeterminate: PropTypes.bool,
+  onChange: PropTypes.func.isRequired,
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  a11yText: null,
+  children: null,
+  isChecked: false,
+  isDisabled: false,
+  isIndeterminate: false,
+  size: ShirtSizes.MEDIUM,
+};
+
+const Checkbox = props => {
+  const checkboxId = React.useRef(uuid());
+  const inputRef = React.useRef(null);
```

Comment:
How about adding `.current` on the end of these two lines, then removing that everywhere else to clean it up a bit?